### PR TITLE
Updated the Sus command.

### PR DESCRIPTION
**Pull request by jayesh-srivastava**
Fixes Issue #710
Updated the sus command and renamed  `SusList.ts` to `susList.ts`.

### CODE REVIEW

File: src/config/commands/susList.ts [hunk 1]
```diff
@@ -32,4 +34,6 @@ export const SusColours = [
   "#928776",
   "#738491",
   "#efc3cf",
+  "#ff7f50",
+  "#ffe135",
```
Comment (nhcarrigan) [inline src/config/commands/susList.ts:38]:
```suggestion
  "#d76464",
  "#fffebe", 
```

These are the hex codes I grabbed from the image.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
A quick suggestion on the hex colours.

You will also need to update the import statement in the command, as indicated in the linked issue.

Finally, you've deleted the pull request template from your PR, which includes important information such as updating the bot version (use `npm version patch` for this one) and indicating whether the documentation page needs an update too (for this PR it does not).

File: src/config/commands/susList.ts (hunk 1 above)
Comment (jayesh-srivastava) [reply src/config/commands/susList.ts:38]: Sure @nhcarrigan I will use the following hex codes and will make an PR which will include the update of the import statement. For now, you can close this PR, gonna make another one.

Comment (nhcarrigan) [reply src/config/commands/susList.ts:38]: You do not need to make a new PR, simply push additional commits to your `patch-1` branch and this PR will automatically update.

Comment (jayesh-srivastava) [reply src/config/commands/susList.ts:38]: @nhcarrigan Okay yeah sure.

Comment (jayesh-srivastava) [reply src/config/commands/susList.ts]: Hey @nhcarrigan I have made the additional changes to the `patch-1` branch. Do I also need to modify the Pull request template?

File: src/config/commands/susList.ts [hunk 2]
```diff
@@ -32,4 +34,6 @@ export const SusColours = [
   "#928776",
   "#738491",
   "#efc3cf",
+  "#d76464",
+  "#fffebe",
```
Comment (nhcarrigan) [inline src/config/commands/susList.ts:38]:
```suggestion
  "#fffebe",
```
Extra space here.

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**